homeassistant/components/arcticspa/const.py

```diff
@@ -0,0 +1,13 @@
+"""Constants for the Arctic Spa integration."""
+
+DOMAIN = "arcticspa"
+
+# How often shall we poll for status on the remote endpoint. (in seconds)
+#
+# The ArcticSpas remote endpoint has a low tolerance for regular polling.
+# (About 15 requests per minute or "every 4 seconds".)
+# It limits with a "429 Too Many Requests" error after a one-minute timeframe.
+UPDATE_INTERVAL = 6.5
```

Comment:
Is there a reason for such a low update interval? Is 10 or 15 seconds not enough?

Reply:
Good point, I think I just wanted a more responsive interface during testing. I'm raising it to 15 seconds in the next commit.